src/components/collection/Collection.js

```diff
@@ -86,8 +104,25 @@ const Collection = ({ id }) => {
           isLoading={isLoading}
         />
         <Divider className={classes.divider} />
-        <Items parentId={id} />
-        <Divider className={classes.divider} />
+        {type === ITEM_TYPES.FOLDER ? (
+          <>
+            <Items parentId={id} />
+            <Divider className={classes.divider} />
+          </>
```

Comment:
Would we want a link from the folder as well to see all the children at once in player?